mergeforward.go

```diff
@@ -22,42 +23,77 @@ var outputdir = flag.String("outputdir", ".", "The output directory of the log f
 type conf interface{}
 
 func main() {
+
+	currentDirectory, err := filepath.Abs(filepath.Dir(os.Args[0]))
+	if err != nil {
+		log.Fatal(err, "- This error happened while automatically detecting the current directory of mergeforward")
+	}
+
+	fmt.Print("Printing current directory: ")
+	fmt.Println(currentDirectory)
+
+	logger, err := seelog.LoggerFromConfigAsString(
+		"<seelog type=\"asynctimer\" asyncinterval=\"1000000\">" +
```

Comment:
try using ` instead of escaping all these characters.  `` means "literal string" and it cleans this sort of thing up immensely.